enterprise/security/src/main/java/org/neo4j/server/security/enterprise/auth/AuthProcedures.java

```diff
@@ -57,30 +57,37 @@
     public KernelTransaction tx;
 
     @Procedure( name = "dbms.security.createUser", mode = DBMS )
-    public void createUser( @Name( "username" ) String username, @Name( "password" ) String password,
-            @Name( "requirePasswordChange" ) boolean requirePasswordChange )
+    public void createUser(
+            @Name( "username" ) String username,
+            @Name( "password" ) String password,
+            @Name( value = "requirePasswordChange", defaultValue = "true"  ) boolean requirePasswordChange
+    )
             throws InvalidArgumentsException, IOException
     {
         EnterpriseAuthSubject adminSubject = ensureAdminAuthSubject();
         adminSubject.getUserManager().newUser( username, password, requirePasswordChange );
     }
 
     @Procedure( name = "dbms.security.changeUserPassword", mode = DBMS )
-    public void changeUserPassword( @Name( "username" ) String username, @Name( "newPassword" ) String newPassword )
+    public void changeUserPassword(
+            @Name( "username" ) String username,
+            @Name( "newPassword" ) String newPassword,
+            @Name( value = "requirePasswordChange", defaultValue = "true" ) boolean requirePasswordChange
+    )
             throws InvalidArgumentsException, IOException
     {
         EnterpriseAuthSubject enterpriseSubject = EnterpriseAuthSubject.castOrFail( authSubject );
         if ( enterpriseSubject.doesUsernameMatch( username ) )
         {
-            enterpriseSubject.setPassword( newPassword );
+            enterpriseSubject.setPassword( newPassword, false );
```

Comment:
No way to find out whether a `true` is default or explicit at this point right? I think it's fair game to use the actual `requiredPasswordChange` then. If you want to change your own password in a smooth way, you would use `changePassword` instead.